app/assets/stylesheets/layouts.css.scss

```diff
@@ -1,6 +1,6 @@
 .body{
   height: 100%;
-  font-family: arial, Georgia, serif;
+  font-family: "American Typewriter", Georgia, serif;
```

Comment:
This is a specific font that many people probably won't have on their computer! You need to add it to the app using `@font-face` in your css file